types/enums/PlanningConstraints.ts

```diff
@@ -12,6 +11,10 @@ export const PlanningConstraints = {
   'designated.nationalPark.broads': 'National Park - Broads',
   'designated.SPA': 'Special Protection Area (SPA)',
   'designated.WHS': 'UNESCO World Heritage Site or buffer zone',
+  flood: 'Flood Risk Zone',
+  'flood.zone.1': 'Flood Risk Zone 1 - Low risk',
+  'flood.zone.2': 'Flood Risk Zone 2 - Medium risk',
+  'flood.zone.3': 'Flood Risk Zone 3 - High risk',
```

Comment:
Optimistically including as we plan to query this dataset in near future: https://trello.com/c/a0Idx2wK/2657-add-flood-risk-zones-from-planning-data-to-list-of-datasets-queried-via-planning-constraints-check